aws-iot-mitigationaction/aws-iot-mitigationaction.json

```diff
@@ -0,0 +1,239 @@
+{
+    "typeName": "AWS::IoT::MitigationAction",
+    "description": "Mitigation actions can to take actions to mitigate issues that were found during an audit",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-iot.git",
+    "definitions": {
+        "Tag": {
+            "description": "A key-value pair to associate with a resource.",
+            "type": "object",
+            "properties": {
+                "Key": {
+                    "type": "string",
+                    "description": "The tag's key.",
+                    "minLength": 1,
+                    "maxLength": 128
+                },
+                "Value": {
+                    "type": "string",
+                    "description": "The tag's value.",
+                    "minLength": 1,
+                    "maxLength": 256
+                }
+            },
+            "required": [
+                "Value",
+                "Key"
+            ]
+        },
+        "ActionParams": {
+            "type": "object",
+            "properties": {
+                "AddThingsToThingGroupParams": {
+                    "$ref": "#/definitions/AddThingsToThingGroupParams"
+                },
+                "EnableIoTLoggingParams": {
+                    "$ref": "#/definitions/EnableIoTLoggingParams"
+                },
+                "PublishFindingToSnsParams": {
+                    "$ref": "#/definitions/PublishFindingToSnsParams"
+                },
+                "ReplaceDefaultPolicyVersionParams": {
+                    "$ref": "#/definitions/ReplaceDefaultPolicyVersionParams"
+                },
+                "UpdateCACertificateParams": {
+                    "$ref": "#/definitions/UpdateCACertificateParams"
+                },
+                "UpdateDeviceCertificateParams": {
+                    "$ref": "#/definitions/UpdateDeviceCertificateParams"
+                }
+            }
+        },
+        "AddThingsToThingGroupParams": {
+            "description": "Allows adding things that triggered the mitigation action to thing groups",
```

Comment:
Minor: let's stay consistent and add periods at the end of all descriptions.

Reply:
Done